Source/Fuse.Scripting.JavaScript/RootableScriptModule.uno

```diff
@@ -53,7 +56,10 @@ namespace Fuse.Scripting.JavaScript
 
 		protected override void CallModuleFunc(Context context, Function moduleFunc, object[] args)
 		{
-			_classInstance.CallMethod(context, moduleFunc, args);
+			if (_classInstance != null)
+				_classInstance.CallMethod(context, moduleFunc, args);
+			else
+				moduleFunc.Call(context, args);
```

Comment:
This also seems bogus to me. This would only trigger if `_names` is null or `CallModuleFunc` was called before `EnsureClassInstanceRooted`, neither seems very reasonable to me.

Reply:
I don't think _names is allowed to be null anymore, but I think it was in the past. I suspect it'd be a good improvement to check it in the constructor(s).

Reply:
Yeah, that's indeed what we should do.

Reply:
@mortoray In `ModelJavaScript`, we're passing in `null` as the NameTable for the JavaScript instance (which then gets passed to RootableScriptModule):

```
//app-level model does not have a nametable otherwise migration would not be possible
var js = new ModelJavaScript(null, modulePath, previewStateId);
```

Any idea what to do there instead then?

Reply:
It'd probably be cleaner to pass in an empty NameTable. Then the code wouldn't need to deal with the null. It's at the root as well, thus it won't cost any unintended overhead. I don't thinks it's an essential change prior to merge though.